<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>855</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1094</width>
    <height>855</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sloth MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../logo/xraysloth_logo_03.svg</normaloff>../../logo/xraysloth_logo_03.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>1091</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="tabLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabData1D">
        <attribute name="icon">
         <iconset>
          <normaloff>../icons/line_chart.svg</normaloff>../icons/line_chart.svg</iconset>
        </attribute>
        <attribute name="title">
         <string>Data 1D</string>
        </attribute>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>381</width>
           <height>591</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="toolboxLayout">
          <item>
           <widget class="QToolBox" name="toolbox1">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="pageSpec">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>379</width>
               <height>531</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Spec file settings</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="pageLoadedData">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>379</width>
               <height>531</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Loaded Data</string>
             </attribute>
             <widget class="QListView" name="listViewData">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>371</width>
                <height>521</height>
               </rect>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>10</y>
           <width>691</width>
           <height>571</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="plotLayout"/>
        </widget>
       </widget>
       <widget class="QWidget" name="tabData2D">
        <attribute name="icon">
         <iconset>
          <normaloff>../icons/contour_chart.svg</normaloff>../icons/contour_chart.svg</iconset>
        </attribute>
        <attribute name="title">
         <string>Data 2D</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabRaytracing">
        <attribute name="title">
         <string>Raytracing</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabRowland">
        <attribute name="title">
         <string>Rowland</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabPeriodicTable">
        <attribute name="icon">
         <iconset>
          <normaloff>../icons/periodic_table.svg</normaloff>../icons/periodic_table.svg</iconset>
        </attribute>
        <attribute name="title">
         <string>Periodic Table</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1081</width>
           <height>441</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="periodicTableLayout">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>449</y>
           <width>521</width>
           <height>141</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="periodicTableInfoLayout">
          <item>
           <widget class="QListWidget" name="ptInfoListWidget"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>449</y>
           <width>551</width>
           <height>141</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QListWidget" name="listWidget_2"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>639</y>
      <width>621</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="consoleLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>639</y>
      <width>461</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="consoleInfoLayout">
     <item>
      <widget class="QTabWidget" name="infoTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabInfoConsole">
        <attribute name="title">
         <string>Console Infos</string>
        </attribute>
        <widget class="QListWidget" name="tabInfoListWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>231</width>
           <height>121</height>
          </rect>
         </property>
        </widget>
        <widget class="QListWidget" name="slothInfoListWidget">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>20</y>
           <width>211</width>
           <height>121</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>231</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Loaded namespaces:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>238</x>
           <y>0</y>
           <width>211</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Loaded Sloth modules/functions:</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabInfo2">
        <attribute name="title">
         <string>Other Infos</string>
        </attribute>
        <widget class="QListView" name="tabInfo2ListView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>481</width>
           <height>141</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit (C-q)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
